.travis.yml

```diff
@@ -3,5 +3,9 @@ language: node_js
 node_js:
   - "4"
   - "6"
-after_success:
-  - npm run codecov
+  - "8"
+before_install: npm install -g npm@latest
```

Comment:
Is npm update needed?

Reply:
Yep, so that npm@5 or more is installed (only the latest major version is supported)

Reply:
The requirements of this package include Node.js `^4.0.0` <sup>[1]</sup>. I think this implies this package expects to work with the npm that Node.js 4.0.0 bundles. If the upgrade of npm is needed, I cannot merge this PR.

[1]: https://github.com/mysticatea/eslint-plugin#requirements

Reply:
I guess the other option is yarn. Without a lock file of some sort a problem like this will reoccur

Reply:
The root cause of the problem is that tests included invalid syntax. The failing is natural. I think I should run CI regularly.

Reply:
You can configure that with Travis Cron beta

Reply:
Thank you! I will configure it later.

Reply:
The other option is for a master only set of tests that delete the package lock before install. This way PRs are not effected but you still get to know about bugs